pg-ragtime/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+# Local pg-ragtime postgresql database
+#
+# I wasn't able to use the toplevel `docker-compose.yaml` out of the box.
+#
+# Maybe `.docker/postgres/initdb.d` is missing?
+#
+# Anyway, I'm adding a local compose file for now to get a REPL to run my
+# code.
+#
+# We probably want to delete this file when I understand how to run the
+# toplevel docker-compose.yaml.
+#
+# Connect with `psql`:
+#
+#     $ psql -d "postgresql://localhost:10170/test?user=test&password=test"
+
+
+version: "3.6"
+
+services:
+  pg16-ragtime:
+    image: postgres:16
+    hostname: pg16-ragtime
+    ports:
+      - "10170:5432"
+    environment:
+      - POSTGRES_DB=test
+      - POSTGRES_PASSWORD=test
+      - POSTGRES_USER=test
```

Comment:
I wasn't able to use the toplevel `docker-compose.yaml`, so I made my own for now.

We probably want to delete this file once I can get the toplevel `docker-compose.yaml` working.

Right now, it fails on my system with this message:

```
$ pwd
/Users/teodorlu/dev/igrishaev/pg2
$ docker-compose up
[+] Running 0/6
 ⠋ Container pg2-pg11-1  Creating                                                                           0.1s 
 ⠋ Container pg2-pg12-1  Creating                                                                           0.1s 
 ⠋ Container pg2-pg14-1  Creating                                                                           0.1s 
 ⠋ Container pg2-pg13-1  Creating                                                                           0.1s 
 ⠋ Container pg2-pg16-1  Creating                                                                           0.1s 
 ⠋ Container pg2-pg15-1  Creating                                                                           0.1s 
Error response from daemon: make cli opts(): making volume mountpoint for volume /Users/teodorlu/dev/igrishaev/pg2/.docker/postgres/initdb.d: mkdir /Users: operation not permitted
```

Reply:
Hm, that's a bit strange. Do you use Linux, by the way? It might be related to permissions with that hidden .docker directory.

Reply:
Another idea: what is your docker version?

Reply:
I'm on Mac. I didn't think about this -- but the fact that I'm using Podman might be causing problems.

```
$ uname -a
Darwin Locomotive 23.2.0 Darwin Kernel Version 23.2.0: Wed Nov 15 21:53:18 PST 2023; root:xnu-10002.61.3~2/RELEASE_ARM64_T6000 arm64
$ docker --version
podman version 4.9.2
$ docker-compose --version
Docker Compose version 2.24.5
$ which docker
docker: aliased to podman
$ which podman
/opt/homebrew/bin/podman
```

I'm not sure exactly how Docker-compose runs, but I assume it's running with Podman, as I don't have a "normal Docker" installed.

Reply:
What versions are you on? (Asking because I assume those are working)